content/docs/studio/user-guide/model-registry/download-models.md

```diff
@@ -0,0 +1,28 @@
+# Download models
+
+If your model file is DVC-tracked, you can download any of its registered
+versions using [dvc get](https://dvc.org/doc/command-reference/get).
+
+```cli
+$ dvc get ${GIT_REPO} ${MODEL_PATH} --rev ${GIT_REV}
+```
+
+For example, one of the models in the
+[bank customer churn demo project](https://github.com/iterative/demo-bank-customer-churn)
+is `lightgbm-model`. You can find its registered versions in the
+[list of Git tags](https://github.com/iterative/demo-bank-customer-churn/tags).
+
+Suppose you want to dowload the model file for the `lightgbm-model@v2.0.1`
+version. In the model details page, you can see that the model file path is
+`.mlem/model/lightgbm-model`.
+
+To download the file, you can use the following dvc command:
+
+```cli
+$ dvc get https://github.com/iterative/demo-bank-customer-churn \
```

Comment:
@alex000kim Can public unauthenticated users get the model file for this project?

Reply:
@tapadipti, looks like not:
```
$ dvc get https://github.com/iterative/demo-bank-customer-churn \ 1.219s
     .mlem/model/lightgbm-model \
     --rev lightgbm-model@v2.0.1
ERROR: unexpected error - [Errno 22] Bad Request: An error occurred (400) when calling the HeadObject operation: Bad Request

Having any troubles? Hit us up at https://dvc.org/support, we are always happy to help!
```

this was an issue before https://github.com/iterative/demo-bank-customer-churn/issues/65

it looks like it's needed to open bucket for `read` and `list` access (I assume)

Reply:
ok, so it was the case 
<img width="204" alt="image" src="https://user-images.githubusercontent.com/6797716/224010642-620319de-5c6a-438e-a10a-710d4ace52bf.png">

Reply:
Let's discuss this in #infrastructure-support